artemis-core-client/src/main/java/org/apache/activemq/artemis/core/protocol/core/impl/ActiveMQSessionContext.java

```diff
@@ -643,7 +643,7 @@ protected CreateSessionMessage newCreateSession(String username,
                                                    boolean autoCommitSends,
                                                    boolean autoCommitAcks,
                                                    boolean preAcknowledge) {
-      return new CreateSessionMessage(name, sessionChannel.getID(), VersionLoader.getVersion().getIncrementingVersion(), username, password, minLargeMessageSize, xa, autoCommitSends, autoCommitAcks, preAcknowledge, confirmationWindow, null);
```

Comment:
There's another commi I had to do on master to fix checkstyle.


if you could please try building with:

```
mvn install -Pdev install -DskipTests=true
```


You will see where the checkstyle failed.


Please amend and a push -f..


I have seen a Pull request for the wildfly fork, you will have to do the same now.